packages/nextjs/hooks/scaffold-eth/useDeployedContractInfo.ts

```diff
@@ -5,7 +5,7 @@ import { usePublicClient } from "wagmi";
 import { Contract, ContractCodeStatus, ContractName, contracts } from "~~/utils/scaffold-eth/contract";
 
 /**
- * Gets the matching contract info from the contracts file generated by `yarn deploy`
+ * Gets the matching contract info from the contracts present in deployedContracts.ts and externalContracts.ts
  * @param contractName - name of deployed contract
```

Comment:
I think it's better to also delete this param

Reply:
yup done 🙌 at https://github.com/scaffold-eth/scaffold-eth-2/pull/665/commits/beb5659bc39a8d5718deffa73121d09c7ae4f472